resource_catalogue_fastapi/utils.py

```diff
@@ -202,61 +342,26 @@ def execute_order_workflow(
     payload = {
         "inputs": {
             "workspace": user_workspace,
+            "workspace_bucket": workspace_bucket,
             "commercial_data_bucket": commercial_data_bucket,
+            "pulsar_url": pulsar_url,
             "product_bundle": product_bundle,
             "stac_key": stac_uri,
         }
     }
+    # Optional inputs that must exist in some form
     if coordinates:
         payload["inputs"]["coordinates"] = str(coordinates)
     else:
         payload["inputs"]["coordinates"] = "[]"
+    # Airbus specific. Should be enforced already.
     if end_users is not None:
```

Comment:
Possibly not correct but would this work instead?
```suggestion
    if end_users:
```